sources/nz-linz.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "", 
+    "edtf:inception": "2020-09-01", 
+    "fullname": "Land Information New Zealand", 
+    "id": 1729181221, 
+    "key": "", 
+    "license": "https://data.linz.govt.nz/layer/104830-fire-and-emergency-nz-localities/", 
```

Comment:
Is there a generic license for all LINZ data? This license text is specific. So Either make the source full anme and prefix specific, or generalize the license text?